internal/clients/nightfall/nightfall_test.go

```diff
@@ -302,6 +302,72 @@ func TestBlurContent(t *testing.T) {
 	}
 }
 
+func TestFilterFileDiffs(t *testing.T) {
+	filePaths := []string{"path/secondary_path/file.txt", "a.go", "a/a.go", "test.go", "path/main.go", "path/test.py"}
+	var fileDiffs []*diffreviewer.FileDiff
```

Comment:
nit - since we know the size of `filePaths` we can make an array of the same size for `fileDiffs`